### PR TITLE
[DE] Translations Deckbuilding

### PR DESCRIPTION
**Pull request by DerSchlupf**
German translation deckbuilding section
image
image

Printable:
image

### CODE REVIEW

**Comment by DerSchlupf** [comment]
i will reviewing and shorting it in the next days

**Comment by DerSchlupf** [comment]
Heyho, 
some questions: 
1) Do you think its possible to move the coin to the first pages. The gap seems big enough.
2) my chosen headline is too long for the content table. I've seen a version, where it is in two lines. what have i to do for these "two-line-trick".

Other than that i have to improve the wording on point 6 of general cardrules. But i feel, merging this isnt so far^^

**Comment by qwrtln** [comment]
>     2. my chosen headline is too long for the content table. I've seen a version, where it is in two lines. what have i to do for these "two-line-trick".

No, currently it's not possible to use two lines. Where did you see it? No translation has it. Are you absolutely sure the title needs to be that long? In English, it's just "Deck-building", not "Deck-building and Card Management" (it wouldn't fit either, BTW).

**Comment by qwrtln** [comment]
To fit the coin, you need to bend some rules:
```po
msgstr ""
"\\vfill\n"
"\\hfill\\includegraphics[width=0.68\\linewidth]{\\art/coin.png}\n"
"\n"
```

File: translations/deckbuilding.tex/de.po [hunk 1]
```diff
@@ -302,11 +430,14 @@ msgid ""
...
 msgstr ""
+"{ \\centering \\includegraphics[width=0.6\\linewidth]{\\cards/specialty."
+"png}\\\\ \\medskip \\footnotesize\\textit{Sonderfähigkeitskarte Stufe IV von "
+"Catherine, der Ritterin} }"
 
 #. type: subsection{#2}
 #: sections/deckbuilding.tex:176
 msgid "Searching Example"
-msgstr ""
+msgstr "Beispiel einer Durchsuchen-Aktion"
```
Comment (qwrtln) [inline translations/deckbuilding.tex/de.po:440]:
```suggestion
msgstr "Beispiel einer\\\\Durchsuchen-Aktion"
```
Maybe to avoid weird spacing.

**Comment by DerSchlupf** [comment]
I see it here: https://discord.com/channels/1257428163105919098/1257428601129926657/1298594653758816257

Deckbuilding In german would be "Deckbau", which isnt a usually used Word in my opiniom. Playerdecks is better. Or just card Management. I will think about a shorter title 👍

**Comment by DerSchlupf** [comment]
Okay, i think its short before merging. I must add an empty space in the headline and because of having four pages here, i want to read them again in the next days. until then, feel free to add suggestions.

**Comment by qwrtln** [comment]
@DarthGandalf This merge conflict needs your attention.

**Comment by qwrtln** [comment]
Yes, and it is showing a merge conflict, one you could have trouble resolving.

**Comment by DarthGandalf** [comment]
This merge is actually trivial, because #760 is not merged yet

**Comment by DarthGandalf** [comment]
Done, and manually marked the outdated texts as fuzzy, since git merge doesn't track that properly

**Review by qwrtln — APPROVED** [review]

**Pull request merged**